app/services/init_monero_client.rb

```diff
@@ -0,0 +1,15 @@
+class InitMoneroClient
+
+  extend LightService::Action
+  promises :monero_client
+
+  executed do |c|
+    RPC.config.host = ENV["XMR_HOST"]
```

Comment:
Is there a way to set this up that doesn't change the global value of `RPC.config`? I hope to find a thread-safe version.

Reply:
I'll take a look at how `Ethereum:HttpClient` does its RPC calls and perhaps adapt its structure to the gem responsible for the monero RPC calls. Thank you for reviewing, I appreciate it!